prql-python/src/lib.rs

```diff
@@ -11,7 +11,7 @@ pub fn compile(prql_query: &str, options: Option<CompileOptions>) -> PyResult<St
         .and_then(prql_compiler::pl_to_rq)
         .and_then(|rq| prql_compiler::rq_to_sql(rq, &options.map(|o| o.into()).unwrap_or_default()))
         .map_err(|e| e.composed("", prql_query, false))
-        .map_err(|e| (PyErr::new::<exceptions::PySyntaxError, _>(e.into_only().unwrap().reason)))
+        .map_err(|e| (PyErr::new::<exceptions::PySyntaxError, _>(e.to_string())))
```

Comment:
@aljazerzen does this look reasonable to you?

With the existing code, I think we would have panicked if there had been more than one `ErrorMessage`. Is just putting the whole thing into a string fine?

Reply:
Yes, looks good.

This is a change in API - no formally since we still return a string, but it's much larger formatted string now.

In any case, we should return the whole error here, maybe serialized to JSON as we do for js.